.github/ISSUE_TEMPLATE/release.yaml

```
@@ -0,0 +1,96 @@
+name:  Release
+description: Release/Pre-release of a version.
+labels: [release]
+body:
+- type: dropdown
+  id: release_type
+  label: Release Type
+  description: which type of release is this release?
+  options:
+  - major release
+  - minor release
+  - patch release
+- type: textarea
+  id: release_version
+  attributes:
+    label: Release Version
+    description: Version of this release (e.g `2.4.0`)
+    placeholder:
+- type: textarea
+  id: release_tests
+  attributes:
+    label: Release Tests
+    description: Github Workflow Test Matrix Checkup
+    placeholder: |
+      **For all releases**
+
+      We maintain some integration tests with 3rd party components which we need to manually verify and update before cutting any release.
+      - [ ] check the testing workflow (`.github/workflows/test.yaml`) and ensure that all matrix versions (`.github/workflows/e2e.yaml` and `.github/workflows/release.yaml`) are up to date for various component releases. If there have been any new releases (major, minor or patch) of those components since the latest version seen in that configuration make sure the new versions get added before proceeding with the release. Remove any versions that are no longer supported by the environment provider.
```

Perhaps we can makes those as links?

```suggestion
      - [ ] check the testing workflow ([`.github/workflows/test.yaml`](/.github/workflows/test.yaml)) and ensure that all matrix versions ([`.github/workflows/e2e.yaml`](/.github/workflows/e2e.yaml) and [`.github/workflows/release.yaml`](/.github/workflows/release.yaml)) are up to date for various component releases. If there have been any new releases (major, minor or patch) of those components since the latest version seen in that configuration make sure the new versions get added before proceeding with the release. Remove any versions that are no longer supported by the environment provider.
```

---

OK, I'll use links for that.